cli/src/vote.rs

```diff
@@ -80,14 +80,22 @@ impl VoteSubCommands for App<'_, '_> {
                         .index(1)
                         .value_name("VOTE_ACCOUNT_ADDRESS")
                         .required(true),
-                        "Vote account in which to set the authorized voter. "),
+                        "Vote account in which to set the authorized voter."),
```

Comment:
Fyi, I think the new `pubkey!` macro expects these spaces.

Reply:
Or I should say, this text will run together without them

Reply:
Oh weird, we couldn't hide that in the macro?

I'll restore them here though

Reply:
I assume we could